guide/de/ch_oview.xml

```diff
@@ -214,7 +214,7 @@
     </sect2>
 
     <sect2 id="oview-featuresbus2">
-      <title>Unterstützung bei der geschäftliche Buchführung</title>
+      <title>Unterstützung bei der geschäftlichen Buchführung</title>
```

Comment:
My German is very rusty, but I would have thought this should become "geschäftlicher" ? I'll leave it to my German colleagues to evaluate this PR....